package.json

```diff
@@ -87,6 +87,9 @@
     "typedoc-plugin-markdown": "^3.11.6",
     "typescript": "^4.3.2"
   },
+  "resolutions":{
```

Comment:
Why is the resolution required, given this is a direct dependency using the same version?

Reply:
without the resolution, somewhere big.js 5.1.0 is still imported